### PR TITLE
Tighten return parameter types within AP_RTC

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

**Comment by rmackay9** [comment]
This has been tested?  I think in general PRs should have a comment about what testing has been done

**Comment by peterbarker** [comment]
@rmackay9 I wrote an example script specifically to test changed to these functions - I merged that yesterday, and this PR contains a patch to accommodate the changes within the library.

The example was run on Linux and on a Pixhawk1, ensuring that the output was identical before/after the patches.

**Comment by rmackay9** [comment]
@peterbarker, great.  I think that's the kind of thing that should be written in the description of the PR.

**Comment by peterbarker** [comment]
On Fri, 16 Aug 2019, Randy Mackay wrote:

> @peterbarker, great. I think that's the kind of thing that should be written in the description of the PR.

It would've been more evident had I not merged the test I wrote 
ahead-of-time :-)

File: libraries/AP_RTC/AP_RTC.cpp [hunk 1]
```diff
@@ -95,8 +95,12 @@ bool AP_RTC::get_system_clock_utc(uint8_t &hour, uint8_t &min, uint8_t &sec, uin
     return true;
 }
 
-// get milliseconds from now to a target time of day expressed as hour, min, sec, ms
-// match starts from first value that is not -1. I.e. specifying hour=-1, minutes=10 will ignore the hour and return time until 10 minutes past 12am (utc)
+// get milliseconds from now to a target time of day expressed as
+// hour, min, sec, ms match starts from first value that is not
```
Comment (OXINARF) [inline libraries/AP_RTC/AP_RTC.cpp]: ms __.__ Match

Comment (peterbarker) [reply libraries/AP_RTC/AP_RTC.cpp]: Fixed, thanks.

**Pull request merged**